unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BotsController/BotsController.cs

```diff
@@ -45,12 +44,12 @@ private IEnumerator EnsureGlobalSceneAndCatalog(bool randomCollections = false)
 
             globalScene = Environment.i.world.state.GetGlobalScenes().First();
 
-            CatalogController.wearableCatalog.Clear();
+            Environment.i.serviceLocator.Get<IWearablesCatalogService>().WearablesCatalog.Clear();
```

Comment:
you could lazy cache both usages in this class: `wearableCatalogService ??= Environment.i.serviceLocator.Get<IWearablesCatalogService>()`

Reply:
Done: https://github.com/decentraland/unity-renderer/pull/4292/commits/01bb62e31f874437d462e6fa417f7358cba7940d